### PR TITLE
fix schema generation for fields defined using ``typing.Any`` #237

### PR DESCRIPTION
**Pull request by Gr1N**
#237

### CODE REVIEW

File: pydantic/fields.py [hunk 1]
```diff
@@ -184,6 +184,10 @@ def _singleton_schema(self, by_alias):
                     'type': 'any_of',
                     'types': [sf.type_schema(by_alias) for sf in self.sub_fields]
                 }
+        elif self.type_ is Any:
+            return {
```
Comment (samuelcolvin) [inline pydantic/fields.py]: wait, shouldn't this just be `return 'any'` same as `return display_as_type(self.type_)`?

Comment (Gr1N) [reply pydantic/fields.py]: agree

**Comment by samuelcolvin** [comment]
Can you also uprev and set the history date to today and we can deploy as soon as this is merged.

**Comment by Gr1N** [comment]
> Can you also uprev and set the history date to today and we can deploy as soon as this is merged.
Let's do this as a separate commit. I can handle it.

**Comment by samuelcolvin** [comment]
ok

**Comment by samuelcolvin** [comment]
looks great, thank you.

**Pull request merged**